### PR TITLE
🐛  import subscribers

### LINKED ISSUE #7748: Import of backup JSON does not restore subscribers.
### Issue Summary

For full blog JSON backup. I have not tried the subscribers-only CSV export/import feature.

### Steps to Reproduce

1. Before upgrading Ghost, I exported a .JSON backup of my data.
2. After a clean install of Ghost.
3. Import the backup JSON.
4. Posts are restored, but subscribers are not.

Here's a redacted screenshot of my backup:

sublime text - don-mccurdy ghost 2016-11-27 json - screen shot nov 26 2016 11 33 09 pm

### Technical details:

* Ghost Version: 0.11.3
* Node Version: v6.9.1
* Browser/OS: Ubuntu 14.04.4 LTS (GNU/Linux 3.13.0-68-generic x86_64)
* Database: ?

### PR DESCRIPTION
**Pull request by kirrg001**
closes #7748

- logic for import of subscribers was just missing

### CODE REVIEW

File: core/server/data/import/data-importer.js [hunk 1]
```diff
@@ -133,6 +133,12 @@ DataImporter.prototype.doImport = function (data) {
                         if (results) {
                             importResults = importResults.concat(results);
                         }
+
+                        return utils.importSubscribers(tableData.subscribers, t);
+                    }).then(function (results) {
+                        if (results) {
+                            importResults = importResults.concat(results);
+                        }
```
Comment (kevinansfield) [inline core/server/data/import/data-importer.js]: There's no `return` of a promise here, will the next `.then` be called successfully?

Comment (kirrg001) [inline core/server/data/import/data-importer.js]: yes 👍 `onFulFilled` callback is resolved with what is returned by it. Bluebird and ES6 Promises behave the same.

Comment (kevinansfield) [inline core/server/data/import/data-importer.js]: Good point, I clearly didn't have my 🤓 on properly there! 😂

**Comment by kirrg001** [comment]
Thanks 👍 I will cherry-pick the change to LTS branch

**Pull request merged**